pdns/recursordist/settings/table.py

```diff
@@ -2154,6 +2154,19 @@
 Throttle a server that has failed to respond :ref:`setting-server-down-max-fails` times for this many seconds.
  ''',
     },
+    {
+        'name' : 'server_down_use_probability',
+        'section' : 'recursor',
+        'type' : LType.Uint64,
+        'default' : '25',
+        'help' : 'Determines the probability of a server marked down to be used anyway',
+        'doc' : '''
+This setting determines the probability of a server marked down to be used anyway.
+A value of ``n`` means that the chance of a server marked down being used after it wins speed selection is is ``1/n``.
+If this setting is zero this mechanism is not active.
```

Comment:
It's not clear to me after reading this sentence whether `0` means that the throttling is never bypassed or that the throttling is never enforced.